src/components/DocsNav.tsx

```diff
@@ -81,50 +120,25 @@ const DocsNav: React.FC<IProps> = ({ relativePath }) => {
   }
 
   // Extract previous and next doc based on current index +/- 1
-  const previousDoc = currentIndex - 1 > 0 ? docsArray[currentIndex - 1] : null
+  const previousDoc =
+    currentIndex - 1 > 0
+      ? { isPrev: true, ...docsArray[currentIndex - 1] }
+      : null
   const nextDoc =
-    currentIndex + 1 < docsArray.length ? docsArray[currentIndex + 1] : null
+    currentIndex + 1 < docsArray.length
+      ? { isNext: true, ...docsArray[currentIndex + 1] }
+      : null
 
   return (
     <Flex
-      direction={{ base: "column-reverse", m: "row" }}
+      as="nav"
+      aria-label="Paginate to document"
+      direction={{ base: "column-reverse", md: "row" }}
       justify="space-between"
-      alignItems={{ base: "center", m: "flex-start" }}
+      alignItems={{ base: "center", md: "flex-start" }}
     >
-      {previousDoc ? (
-        <Card justify="flex-start">
-          <Link to={previousDoc.to} textDecoration="none" p={4} h="100%">
-            <Emoji text=":point_left:" fontSize="5xl" />
-          </Link>
-          <TextDiv ps="0">
-            <Text textTransform="uppercase" m="0">
-              <Translation id="previous" />
-            </Text>
-            <Link to={previousDoc.to} textAlign="start">
-              <Translation id={previousDoc.id} />
-            </Link>
-          </TextDiv>
-        </Card>
-      ) : (
-        <Flex />
-      )}
-      {nextDoc ? (
-        <Card justify="flex-end">
-          <TextDiv alignItems="flex-end" pe="0">
-            <Text textTransform="uppercase" m="0">
-              <Translation id="next" />
-            </Text>
-            <Link to={nextDoc.to} textAlign="end">
-              <Translation id={nextDoc.id} />
-            </Link>
-          </TextDiv>
-          <Link to={nextDoc.to} textDecoration="none" p={4} h="100%">
-            <Emoji text=":point_right:" fontSize="5xl" />
-          </Link>
-        </Card>
-      ) : (
-        <Flex />
-      )}
+      {previousDoc ? <CardLink docData={previousDoc} /> : <Spacer />}
+      {nextDoc ? <CardLink docData={nextDoc} /> : <Spacer />}
```

Comment:
nitpick: instead of changing the datastructure of `docData` I'd handle the `isNext` / `isPrev` as individual props.

```tsx
// you'd have
{previousDoc ? <CardLink docData={previousDoc} isNext /> : <Spacer />}
{nextDoc ? <CardLink docData={nextDoc} isPrev /> : <Spacer />}
```
I think its simpler and clearer than having it nested inside the `docData` object.

Reply:
Makes sense. I'll make that adjustment! 😁